src/App.tsx

```diff
@@ -1,31 +1,71 @@
-import './App.scss';
+import {
+  NavLink,
+  Route,
+  Routes,
+} from 'react-router-dom';
 
-/*
-import { RouteComponentProps } from 'react-router-dom';
+import 'bulma/css/bulma.css';
+import './App.scss';
+import { HomePage } from './components/HomePage/HomePage';
+import { PageInfo } from './components/PageInfo/PageInfo';
+import { tabs } from './api/tabs';
+import { TabsPage } from './components/TabsPage/TabsPage';
+import { TabOutlet } from './components/TabsOutlet/TabsOutlet';
+import { Loader } from './components/Loader';
 
-type TabsPageProps = React.FC<RouteComponentProps<{ tabId: string }>>;
-const TabsPage: TabsPageProps = ({ match }) => {...};
+export const App = () => {
+  const activeLink = ({ isActive }: any) => ({
+    color: isActive ? '#fff' : '#545e6f',
+    background: isActive ? '#7600dc' : '#f0f0f0',
+  });
 
-or
+  return (
+    <div className="App">
+      <header className="App__header">
+        <nav className="tabs is-large App__nav">
+          <NavLink
+            to="/"
+            style={activeLink}
+          >
+            Home
+          </NavLink>
+          <NavLink
+            to="/tabs"
+            style={activeLink}
+          >
+            Tabs
+          </NavLink>
+        </nav>
+      </header>
 
-import { useParams } from 'react-router-dom';
+      <div className="App__container">
+        <Routes>
+          <Route path="/">
+            <Route
+              index
+              element={<HomePage />}
+            />
+          </Route>
 
-const TabsPage = () => {
-  const { tabId } = useParams<{ tabId: string }>();
-  ...
+          <Route
+            path="tabs"
+            element={<TabOutlet />}
+          >
```

Comment:
if you don't have anything inside your parent component, you can render its childs without `Outlet` component
```suggestion
          <Route path="tabs" />
```